src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -73,15 +75,23 @@ public static AuthenticationBuilder AddScriptJwtBearer(this AuthenticationBuilde
 
         private static TokenValidationParameters CreateTokenValidationParameters()
         {
+            List<SymmetricSecurityKey> signingKeys = new List<SymmetricSecurityKey>();
+            string defaultKey = Util.GetDefaultKeyValue();
+            if (defaultKey != null)
+            {
+                signingKeys.Add(new SymmetricSecurityKey(defaultKey.ToKeyBytes()));
+                signingKeys.Add(new SymmetricSecurityKey(Encoding.UTF8.GetBytes(defaultKey)));
+            }
+
+            if (SecretsUtility.TryGetEncryptionKey(out string key) && key != defaultKey)
```

Comment:
Can we move this from `key != defaultKey` to `string.Equals` so we're explicit about casing?

Reply:
Fixed. Using string.Equals default case sensitive